cpp/test/compat/comm/commcompat.cpp

```diff
@@ -15,7 +15,11 @@
 #include "commcmd_arg_types.h"
 #include "commcmd_arg_reflection.h"
 
-#define die(ARGS) (printf ARGS, exit(1), true);
+#define die(ARGS) printf(ARGS); exit(1);
```

Comment:
Could also just turn this into a function like we did [for the core compat tests](https://github.com/Microsoft/bond/blame/7a837b4b9696cd20915b07100d9859acee20e6a2/cpp/test/compat/core/compat.cpp#L20). (It say's you're the author, but I did this before you squash merged.)